src/components/shared/Entity/HeaderLogs.tsx

```diff
@@ -0,0 +1,44 @@
+import { FormattedMessage } from 'react-intl';
+import {
+    useFormStateStore_logToken,
+    useFormStateStore_messagePrefix,
+    useFormStateStore_showLogs,
+    useFormStateStore_status,
+} from 'stores/FormState/hooks';
+import { FormStatus } from 'stores/FormState/types';
+import useEntityWorkflowHelpers from './hooks/useEntityWorkflowHelpers';
+import LogDialog from './LogDialog';
+import LogDialogActions from './LogDialogActions';
+
+// TODO (break logs from buttons) - this is starting to display the dialog as a stand alone
```

Comment:
Assuming this TODO comment identifies a task out of scope for this PR.

Reply:
Yeah... I didn't want to lose this component as it is really nice to help with extra renders but also didn't want an old file floating around.